src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -158,9 +282,49 @@ const GroupCheckbox = styled(Box)`
   }
 `;
 
-const StyledCount = styled(Count)`
+const PrimaryCount = styled(Count)`
+  font-size: 18px;
+  color: ${p => p.theme.gray700};
+`;
+
+const SecondaryCount = styled(({value, ...p}) => (
+  <React.Fragment>
+    {'/'}
+    <Count {...p} value={value} />
+  </React.Fragment>
+))`
   font-size: 18px;
-  color: ${p => p.theme.gray600};
+  color: ${p => p.theme.gray500};
+`;
+
+const TooltipContent = styled(p => (
+  <table {...p}>
```

Comment:
Should you destructure children so it doesn't get passed?

Reply:
Doesn't change the final dom produced, but I will do it for better code quality